docs/reference/SetCache.md

```diff
@@ -0,0 +1,74 @@
+## SetCache ##
+
+```python
+fpdf.set_cache(cache)
+```
+
+### Description ###
+
+This method change caching mechanism for current fpdf class instance.
+If this method not called before [AddFont](AddFont.md) cached files are
+created in same folder with fonts.
+
+**Note**: Prior version `1.7.3` caching behaviour can be changed 
```

Comment:
“Prior to version 1.7.3, the caching behaviour can be changed by setting the FPDF_CACHE_MODE constant”, otherwise I could read it as this was the way to set caching in version <= 1.7.3, rather than in versions < 1.7.3.